### PR TITLE
minor update

### PR DESCRIPTION
**Pull request by awwmicky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ Simple joke making application created with Node.js & Express.js. It will demons
 ## Collaborators
 
 [ **Michael F. Alvarez** ](https://www.linkedin.com/in/awwmicky/) |
- [ **Herman Liu** ](https://www.linkedin.com/in/hermanliu168/) |
```
Comment (sanixstudio) [inline README.md:12]: What the heck is this?

**Review by sanixstudio — CHANGES_REQUESTED** [review]
Fix this

File: README.md (hunk 1 above)
Comment (awwmicky) [reply README.md:12]: no, it's good sun.

**Review by sanixstudio — APPROVED** [review]
I approved it

**Pull request merged**